moduleroot/CONTRIBUTING.md.erb

```diff
@@ -174,6 +174,18 @@ and so on. rspec tests may have the same kind of dependencies as the
 module they are testing. While the module defines in its [Modulefile](./Modulefile),
 rspec tests define them in [.fixtures.yml](./fixtures.yml).
 
+To run specific tests, use the spec test file name and a filter like:
```

Comment:
Would it make sense to link to https://relishapp.com/rspec/rspec-core/v/3-9/docs/command-line/example-option

Reply:
Done